libr/core/visual.c

```diff
@@ -4036,3 +4036,101 @@ R_API int r_core_visual(RCore *core, const char *input) {
 	r_cons_show_cursor (true);
 	return 0;
 }
+
+// TODO: move this to the table api
+R_API void r_core_visual_list(RCore *core, RList *list, ut64 seek, ut64 len, int width, int use_color) {
```

Comment:
keep it in here, until we can move it outside

Reply:
Okay Sir